fuzz.go

```diff
@@ -61,6 +61,27 @@ func NewWithSeed(seed int64) *Fuzzer {
 	return f
 }
 
+// NewFromGoFuzz is a helper function that enables using gofuzz (this
+// project) with go-fuzz (https://github.com/dvyukov/go-fuzz) for continuous
+// fuzzing. Essentially, it enables translating the fuzzing bytes from
+// go-fuzz to any Go object using this library.
+//
```

Comment:
Add a note that, given the intended use, it doesn't make sense to use the resulting Fuzzer in parallel--you'd lose the determinism which is the point of this. And the implementation is therefore not thread safe.